### PR TITLE
#412 set up programs db infrastructure

### PR DESCRIPTION
**Pull request by rrabinovitch**
Co-authored by: SanazDn <sanaz142@yahoo.com>
Co-authored by: Ruthie Rabinovitch <62635544+rrabinovitch@users.noreply.github.com>

## Proposed changes

This PR adds a migration file to set up tables for `curriculums`, `programs`, `activity_types`, `activities`, and `participant_activities`. The migration file also includes examples of `activity_type` records to populate the `activity_types` table.

For `start_date` and `end_date` in the `programs` table, we used the SQL `DATE` type, which uses the `YYYY-MM-DD` format.

For `start_time` and `end_time` in the `activities` table, we used the SQL `TIME` type, which uses the `hh:mm:ss` format. Afternoon times should be represented in 24 hr structure (eg, 3pm would translate to `15:00:00`)

## Checklist

- [x] Are the issues being addressed linked to this PR?
- [x] Do all commit messages start with the issue number?

### CODE REVIEW

**Comment by seidior** [comment]
@rrabinovitch @SanazDn Thanks for creating this pull request! Can you please link this PR to the repository project "Sprint 17" so it shows up on our sprint Kanban board? Thank you.

File: db/migration/V019__create_programs.sql
Comment (seidior) [inline db/migration/V019__create_programs.sql:11]: Let's tighten up this space here before we merge into main.

File: db/migration/V019__create_programs.sql
Comment (seidior) [inline db/migration/V019__create_programs.sql:78]: Please add a trailing space at the end of the file.

**Review by seidior — CHANGES_REQUESTED** [review]
Everything looks good to me. I can approve this once the formatting is modified!

**Review by seidior — APPROVED** [review]

**Comment by seidior** [comment]
Great work on this and fast turnaround. I know we have also tested this code in other repositories and it works just fine, and the migration completes successfully in my local environment. As long as we have other positive reviews, I see no reason that we can't merge and push this live.

File: db/migration/V019__create_programs.sql
Comment (camianderson) [inline db/migration/V019__create_programs.sql:63]: Great job fixing the typo!

**Review by camianderson — APPROVED** [review]
Great work! I didn't find any other issue! So you are good to merge it!

**Review by ChainShoe-Z — APPROVED** [review]
reviewed

**Pull request merged**
